container.fc

```diff
@@ -48,6 +48,7 @@
 /usr/s?bin/crio.*			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/local/s?bin/crio.*			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/s?bin/ocid.*			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
+/usr/bin/prometheus-podman-exporter			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
```

Comment:
Container_runtime_t is a very permissive (unconfined) domain, does this tool basically run podman?

Reply:
> does this tool basically run podman?

`/usr/bin/prometheus-podman-exporter` is this tool, which is essentially a network-facing wrapper around `podman ps` and `podman stats`. 

> Container_runtime_t is a very permissive (unconfined) domain

I can test labelling it with a different domain, do you have any suggestions? This one was giving me bizarre errors (see the commit message for “[Label prometheus-podman-exporter as container_runtime_exec_t](https://github.com/containers/container-selinux/pull/383/commits/729136a1a3ffbb746bd9152ba85a531c8b706046)”), so I have no idea which domains to try.